web/app/room/[roomId]/page.jsx

```
@@ -730,20 +833,19 @@ export default function RoomPage({ params }) {
-              <h3 className="text-lg font-semibold">Flag Submission</h3>
-              <p className="text-xs font-mono text-muted-foreground break-all">
-                {flagging.submission_id}
-              </p>
+              <h3 className="text-lg font-bold">Flag Submission</h3>
               <form onSubmit={onFlagSubmit} className="space-y-4">
                 <div>
-                  <label className="text-sm font-medium">Reason</label>
+                  <label className="text-xs font-black uppercase text-muted-foreground">
+                    Reason
+                  </label>
                   <textarea
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)